src/pages/scheduler/components/share/SelectedCoursesCardList.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Flex, Heading, HStack, VStack, Wrap, WrapItem, Text } from '@chakra-ui/react';
+
+import { useDarkMode } from 'lib/hooks/useDarkMode';
+import { SavedCourse } from 'lib/hooks/useSavedCourses';
+import { getReadableTerm } from 'lib/utils';
+
+type ShareCourseCard_Props = {
```

Comment:
nit: this could be `ShareCourseCardProps`, to avoid mixing camel case and underscores